src/ProgressOnderwijsUtils/Data/SqlParameterComponent.cs

```diff
@@ -127,15 +127,14 @@ internal struct CustomTableType
 
             public static readonly Dictionary<Type, string> SqlTableTypeNameByDotnetType = All.ToDictionary(o => o.Type, o => o.SqlTypeName);
 
-            public static ParameterizedSql DefinitionScripts =>
-                ParameterizedSql.CreateDynamic($@"
-                    set transaction isolation level serializable;
-                    begin tran

```

Comment:
...but you probably should leave this in, since nested explicit transactions are OK, and this way a caller that does not supply a transaction doesn't run into surprises.